woocommerce/wc-admin
# Fix payments extensions displayed below the offline payments options

Fixes #8230

Sort the payment gateways by the "plugins" property to ensure the offline payments options are always below the payments extensions

### Screenshots

Before:

![before](https://camo.githubusercontent.com/180a48233a7310cfed30d3dc729e00ec52efa2c5aa51f7a078c798369cf468dc/68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3564383466653865313230316465303030316236616236312f65343338396265322d383133342d343238342d383837302d376230383737313430313066)

After:

![Screen Shot 2022-01-31 at 15 34 34](https://user-images.githubusercontent.com/4344253/151755981-8d473f92-f1b1-4cfb-925b-a7520ace9c7e.png)



### Detailed test instructions:

1. Create a new store
2. Go to the setup wizard and choose the "Australia" or "India" as **Country / Region**
3. Go to **WooCommerce > Home** and visit the "Set up payments" task
4. Observe that offline payments options are below the payments extensions.
